### DISCUSSION TITLE
Problem with Raid FpS

### DISCUSSION THREAD

**Discussion by Senchay92** [siweia/NDui#195] [Q&A]
Hi everyone, 
i use ndui along time in classic. Now i have a problem. If im in a 25 raid my fps  goes on boss fight down to 2 fps. In a 5 man dugeon i dont have this problem. I put down the settings in raids but nothing helps. 
Today i play Naxx 25 without the addon an my fps was at 58,also okay. 

Have anybody an idea or the same problem. I like ndui but i cant play 25 raids with the addon , that´s crap ;( 

best wishes 
Senchay

**Comment by siweia** [comment]
Do you have the issue when you have NDui only?

**Reply by Senchay92** [reply]
What do you men with ndui only ? 
do you mean i shloud turn off the other addons like dmb and weak auras?

**Reply by siweia** [reply]
Yes. Enable NDui and DBM only, disable all the rest, especially the weakauras.

**Reply by Senchay92** [reply]
Okay yesterday, i turn really all off but not WA. Why do you mean its importent? 
i mean WA and DbM are very importent addons. When this doesnt help do you have an another tip? 

So i have a second question. Is there a way to switch the languagues? In german for example?

**Reply by siweia** [reply]
Most of WA wrote in bad coding, and cause fps drop. I knew DBM should be important,  but not for WA.

No locales for German yet. If you are insterested, you can provide a list of German locale base on English. Which contains in NDui/locales.

**Reply by Senchay92** [reply]
Mmmh okay i will test it. 
It’s a strange because all addons work together very good but ndui Not. And the problem occurred for the first time in tbc BT.
Is ndui finally programmed for woltk classic?

**Reply by siweia** [reply]
I do have someone mentioned fps drops in NAXX, but I don't play classic at all, no way to test this issue myself.
Most people feel smooth any way.

**Comment by Senchay92** [comment]
Hi, 
yes i have the fps problem only if i use the addon and only in 25 men raids. Is there an option to reset the addon or do you have a answer? 
i dont want delet the addon because i really like nudi and want to use it
